src/Utilities/Compiler/WellKnownTypeNames.cs

```diff
@@ -333,6 +333,7 @@ internal static class WellKnownTypeNames
         public const string SystemTextEncoding = "System.Text.Encoding";
         public const string SystemTextRegularExpressionsRegex = "System.Text.RegularExpressions.Regex";
         public const string SystemTextStringBuilder = "System.Text.StringBuilder";
+        public const string SystemTextJsonSerializationJsonConstructorAttribute = "System.Text.Json.Serialization.JsonConstructorAttribute";
```

Comment:
nit: should be before `SystemTextRegularExpressionsRegex` to keep these constants sorted.

Reply:
Thanks, I'd missed the comment that lines should be sorted in this file. Done.